PendingReleaseNotes

```diff
@@ -102,3 +102,15 @@ https://docs.ceph.com/en/quincy/cephadm/upgrade/#staggered-upgrade
 Relevant tracker: https://tracker.ceph.com/issues/55715
 
   Relevant tracker: https://tracker.ceph.com/issues/5614
+  
+* Cephadm may now be configured to carry out CephFS MDS upgrades without
+reducing ``max_mds`` to 1. Previously, Cephadm would reduce ``max_mds`` to 1 to
```

Comment:
Might “new versions” be changed to “different formats”?

Reply:
new versions?

Reply:
Below.  Reviewing PRs on my phone is slightly awkward.